packages/contractkit/src/wrappers/SortedOracles.test.ts

```diff
@@ -115,6 +122,81 @@ testWithGanache('SortedOracles Wrapper', (web3) => {
     })
   })
 
+  describe('#removeExpiredReports', () => {
+    beforeEach(async () => {
+      await reportAsOracles(stableTokenOracles)
+    })
+
+    describe('when expired reports exist', () => {
+      beforeEach(async () => {
+        const expirySeconds = (await sortedOracles.reportExpirySeconds()).toNumber()
+        await timeTravel(expirySeconds + 5, web3)
+      })
+
+      it('should successfully remove a report', async () => {
+        const initialReportCount = await sortedOracles.numRates(CeloContract.StableToken)
+        const tx = await sortedOracles.removeExpiredReports(CeloContract.StableToken, 1)
+        await tx.sendAndWaitForReceipt({ from: oracleAddress })
+        expect(await sortedOracles.numRates(CeloContract.StableToken)).toEqual(
+          initialReportCount - 1
+        )
+      })
+    })
+
+    describe('when reports exist but are not expired', () => {
+      it('should not remove any reports', async () => {
+        const initialReportCount = await sortedOracles.numRates(CeloContract.StableToken)
+        const tx = await sortedOracles.removeExpiredReports(CeloContract.StableToken, 1)
+        await tx.sendAndWaitForReceipt({ from: oracleAddress })
+        expect(await sortedOracles.numRates(CeloContract.StableToken)).toEqual(initialReportCount)
+      })
+    })
+  })
+
+  describe('#isOldestReportExpired', () => {
+    let expirySeconds: number
+
+    beforeEach(async () => {
+      expirySeconds = (await sortedOracles.reportExpirySeconds()).toNumber()
+    })
+
+    describe('when at least one expired report exists', () => {
```

Comment:
ditto?
style choice at this point

Reply:
I think in some of these cases it aids readability - separating the setup with the expectation-checks. The ones I'd like to leave as-is are the ones where the setup is more than a couple lines, and/or the combined text of `describe` + `it` would extend past the line width limit.